platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -303,62 +303,138 @@ static String get_mountpoint(const String &p_path) {
 }
 
 Error OS_LinuxBSD::move_to_trash(const String &p_path) {
-	String trash_can = "";
+	Error ok;
```

Comment:
We don't need to declare variables before they're needed. You can remove this and change what is currently line 311 to `Error ok = execute(` etc.

Also, maybe this variable should be named `err` or `result` or something instead of `ok`. I see that there is also `err` below, I think this code would be a good candidate for adding some scopes using `{}`.

Reply:
No prob, I'll change it :)
Tried to emulate the style of the code already written,
i copied the style from OS_LinuxBSD::shell_open() on line 124.